app/fetcher/hospital_capacities_fetcher.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+
+from app import db, app
+from app.fetcher.fetcher import Fetcher
+from app.models import OckovaniSpotreba, OckovaciMisto, KapacityNemocnic
+
+
+class HospitalCapacitiesFetcher(Fetcher):
+    """
+    Class for updating capacities in hospitals.
+    """
+
+    USED_CSV = 'https://dip.mzcr.cz/api/v1/kapacity-intenzivni-pece-zdravotnicke-zarizeni-10-2021.csv'
+
+    def __init__(self):
+        super().__init__(KapacityNemocnic.__tablename__, self.USED_CSV)
+
+    def fetch(self, import_id: int) -> None:
+        df = pd.read_csv(self._url)
+
+        df['luzka_standard_kyslik_kapacita_volna_covid_pozitivni'] = df[
```

Comment:
it should be possible to use something like this:
```
for col in df.columns:
  if col not in ['blacklisted_column']:
    df[col] = df[col].fillna('0').astype('int')
```

Reply:
Ok, changed